tests/system-tests/parse-php8-code/vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+// autoload.php @generated by Composer
+
+require_once __DIR__ . '/composer/autoload_real.php';
+
+return ComposerAutoloaderInita53ca8a540a790451fc389ced0b5860d::getLoader();
```

Comment:
Not sure what is the goal here, but vendor should not be part of repository. We won't maintain it

Reply:
I understand it looks confusing. But please have a look here: https://github.com/mpdude/rector/tree/system-level-tests/tests/system-tests/parse-php8-code

This directory is a fixture. It feels like a project/codebase. You can run `/path/to/your/bin/rector process` in it. 

The `vendor` is not for real vendors, but the result of having a `composer.json` file and running `composer dump-autoload`. This way, Rector finds everything it expects from a project.

We will never have to run `composer update` or similar in this directory. It's just a "fake" to run a particular test.

Reply:
> The vendor is not for real vendors, but the result of having a composer.json file and running composer dump-autoload. This way, Rector finds everything it expects from a project.

That should be part the build script.